src/startup/routes.jsx

```diff
@@ -1,7 +1,12 @@
 import { HomePage } from 'home'
+import { DetectionPage } from 'detection'
 import { About } from 'about'
 
 export const routes = [{
+  path: '/detection',
```

Comment:
Since this page is meant to represent a single detection, shouldn't the route look something like `/detection/1` ? 

[Here](https://reactrouter.com/web/guides/quick-start)'s a link to their documentation to see how to do that.